src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfMapperUtil.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.et.syaapi.service.pdf;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.et.common.model.ccd.Address;
+
+import java.util.Locale;
+
+public final class PdfMapperUtil {
+
+    private PdfMapperUtil() {
+
+    }
+
+    private static String formatAddressLines(String addressLine) {
+        return toCapitalCase(addressLine).trim();
+    }
+
+    private static String toCapitalCase(String addressLine) {
+        String[] addressLineWords = addressLine.toLowerCase(Locale.UK).split(" ");
+
+        StringBuilder addressLineModified = new StringBuilder();
+        for (String word : addressLineWords) {
+            addressLineModified.append(word.substring(0, 1).toUpperCase(Locale.UK))
+                .append(word.substring(1)).append(' ');
+        }
+        return addressLineModified.toString();
+    }
+
+    public static String convertAddressToString(Address address) {
+        StringBuilder addressStringValue = new StringBuilder();
+        if (StringUtils.isNotEmpty(address.getAddressLine1())) {
+            addressStringValue.append(formatAddressLines(address.getAddressLine1()));
+        }
+        if (StringUtils.isNotEmpty(address.getAddressLine2())) {
```

Comment:
May be check empty string in formatAddressLines() method once rather than for all strings here?

Reply:
Not that I feel ignored (actually I do a bit) but PR shouldn't be approved/merged if there are still any unresolved comments. This was the practice at least in ECM :)